module/programs/helper.js

```diff
@@ -1067,6 +1069,46 @@ module.exports = class ProgramsHelper {
 						result: [],
 					})
 				}
+				if (projectsCount) {
+					//Filtering out all the program IDs
+					let allProjectIds = programsData.flatMap((program) => program._id || [])
```

Comment:
@praveenKDass rename the varable to userProgramIds

Reply:
Changed @VISHNUDAS-tunerlabs